### PR TITLE
[ch53019] add Prolude.Persist

### PR DESCRIPTION
**Pull request by goolord**

### CODE REVIEW

**Comment by tfausak** [comment]
Fixes #12.

File: prolude.cabal [hunk 1]
```diff
@@ -47,4 +48,6 @@ library
         , scientific >= 0.3.6 && < 0.4
         , text >= 1.2.3 && < 1.3
         , time >= 1.9.3 && < 1.10
-        , vector >= 0.12.1 && < 0.13
\ No newline at end of file
+        , vector >= 0.12.1 && < 0.13
+        , persistent >= 2.8 && < 3
+        , persistent-mongoDB >= 2.8 && < 3
```
Comment (tfausak) [inline prolude.cabal:53]: Can we keep these sorted alphabetically?

File: prolude.cabal [hunk 2]
```diff
@@ -43,8 +44,10 @@ library
         , cassava >= 0.5.2.0 && < 0.6
         , containers >= 0.6.2 && < 0.7
         , mongoDB >= 2.7.0 && < 2.8
+        , persistent >= 2.8 && < 3
+        , persistent-mongoDB >= 2.8 && < 3
```
Comment (tfausak) [inline prolude.cabal:48]: I think this bounds should really be something like `>= 2.8.0 && < 2.12`, but they're probably fine as is.

**Review by tfausak — APPROVED** [review]

File: prolude.cabal (hunk 2 above)
Comment (goolord) [reply prolude.cabal:48]: the bound on `persistent` in `persistent-mongoDB` is `>= 2.8 && < 3`, so I figure they will not break their promise of not making any breaking changes until the next major version

**Pull request merged**